gitops/sre/example/credentials/kubeconfig/compute/README.md

```diff
@@ -0,0 +1,14 @@
+# Clusters credentials 
+
+This directory purpose is to contain the kubeconfig 
+files used to access and control the various clusters.
+
+The filename should match the directory name of
+the cluster in the `environment` directory.
+
```

Comment:
no this is not what we have today. The directoy name should match the name of the cluster in the kubeconfig file.